src/yaml/yaml.cr

```diff
@@ -1,5 +1,38 @@
 require "./*"
 
+# The YAML module serves to deserialize data from YAML into accessible structure.
```

Comment:
Well, thinking about it I'd keep it out completely, "This module provides deserialization of YAML to native Crystal datastructures" or something.